services/src/main/scala/cromwell/services/metadata/impl/builder/MetadataComponent.scala

```diff
@@ -146,7 +146,7 @@ case class MetadataList(v: Map[Int, MetadataComponent]) extends MetadataComponen
 object MetadataPrimitive {
   val ExecutionStatusOrdering: Ordering[MetadataPrimitive] = Ordering.by { primitive: MetadataPrimitive =>
     ExecutionStatus.withName(primitive.v.value)
-  }
+  }(ExecutionStatus.ExecutionStatusOrdering)
```

Comment:
2.13 couldn't choose between multiple `Ordering` options so this became explicit